test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -552,14 +552,16 @@ describe('Client Side Encryption Prose Tests', function () {
     const limitsKey = loadLimits('limits-key.json');
     const limitsDoc = loadLimits('limits-doc.json');
 
-    before(function () {
-      // First, perform the setup.
+    let firstTimeSetup = true;
+    beforeEach(async function () {
```

Comment:
could we just use a `before` hook here to run this code once before the tests start?

Reply:
disregard this comment - I didn't see Neal's comment when I reviewed the PR.